tests/test/org/commcare/model/test/ProfileTests.java

```diff
@@ -83,4 +110,23 @@ private Profile getBasicProfile() {
             throw PersistableSandbox.wrap("Error during profile test setup", e);
         }
     }
+
+    // Return a Profile object constructed from a profile.ccpr file explicitly containing the
+    // fields necessary for multiple apps support
+    private Profile getMultipleAppsProfile() {
```

Comment:
This should probably just be the same method as above and take in a string arg that is the path